src/py/mat3ra/made/tools/calculate.py

```diff
@@ -125,3 +132,44 @@ def calculate_interfacial_energy(
     surface_energy_film = calculate_surface_energy(film_slab, film_bulk, calculator)
     adhesion_energy = calculate_adhesion_energy(interface, substrate_slab, film_slab, calculator)
     return surface_energy_film + surface_energy_substrate - adhesion_energy
+
+
+class InteractionCalculatorParameters(BaseModel):
+    shadowing_radius: float = 2.5
```

Comment:
The InteractionCalculatorParameters seems more generic and should not depend on shadowing. But InterfaceInteractionCalculatorParameters could. The parameters should be explained here and then may be referred to on line 152 below